index.js

```diff
@@ -312,6 +312,10 @@ class ResembleHelper extends Helper {
       options.tolerance = 0;
     }
 
+    if (options.ignoredElement !== undefined) {
+      options.ignoredElement = await this._getElementCoordinates(options.ignoredElement);
```

Comment:
here the element is transformed to ignored area. Can't we use the `ignoredBox` property to store the coordinates here?

Reply:
Done.